anaUltraScurve.py

```diff
@@ -249,9 +321,8 @@ def overlay_fit(VFAT, CHAN):
             noise[0] = param1
             fitENC.append(vToQm*param1*options.ztrim)
             pedestal[0] = param2
-            if noise[0] > 20.0 or ped_eff[0] > 50.0: mask[0] = True
```

Comment:
> We also will need to make sure we mask channels which give no response during the scurve.

This corresponds to `maskReason = MaskReason.DeadChannel` (no data at all) or `maskReason = MaskReason.FitFailed` (not enough data to fit).